internal/api/handler/admin_login_v1alpha1.go

```diff
@@ -46,7 +46,7 @@ func AdminLoginV1Alpha1(cfg *config.AdminConfig) AdminLoginV1Alpha1Func {
 			jwt.SigningMethodHS256,
 			jwt.RegisteredClaims{
 				IssuedAt:  jwt.NewNumericDate(now),
-				Issuer:    "kargo",
+				Issuer:    cfg.TokenIssuer,
```

Comment:
Not related to the PR, but I noticed we issue a 1-hour hard-wired expiration time.

While you are here, maybe consider making it configurable with an `ADMIN_ACCOUNT_TOKEN_EXPIRY` env var?

Reply:
I'll submit a follow-up.